index.js

```diff
@@ -59,7 +52,7 @@ async function fastifyRoutesStats (fastify, opts) {
 
   fastify.addHook('onSend', function (request, reply, _, next) {
     if (request[decoratorName]) {
-      const routeId = reply.context.config.statsId || request.raw.url
+      const routeId = request.routeOptions.config.statsId || request.raw.url
```

Comment:
`reply.context` will be deprecated in Fastify v5, thus I used the new suggested `request.routeOptions.config`